src/mappings/lpTokenPriceHandler.ts

```diff
@@ -3,18 +3,25 @@ import { u32 } from '@polkadot/types';
 import { divDecs } from './util';
 
 const relayAssetId = (api.consts.crowdloans.relayCurrency as u32).toNumber();
+const nativeAssetId = api?.consts?.currencyAdapter?.getNativeCurrencyId?.toString();
 
-export const getLptokens = async () => {
+export const getLpTokens = async () => {
   const metadatas = await api.query.assets.metadata.entries();
-  const allAssets = metadatas.map(([{ args }, metadata]) => {
-    const [assetId] = args;
-    const { symbol, decimals } = metadata;
-    return {
-      assetId: assetId.toNumber(),
-      symbol: symbol.toHuman().toString(),
-      decimals: decimals.toNumber()
-    };
-  });
+  const allAssets = [
+    ...metadatas.map(([{ args }, metadata]) => {
+      const [assetId] = args;
+      const { symbol, decimals } = metadata;
+      return {
+        assetId: assetId.toNumber(),
+        symbol: symbol.toHuman().toString(),
+        decimals: decimals.toNumber()
+      };
+    }),
+    {
+      assetId: api?.consts?.currencyAdapter?.getNativeCurrencyId?.toString(),
+      symbol: nativeAssetId === '0' ? 'HKO' : 'PARA' // hard code, cuz rpc is not available in subquery
```

Comment:
same above, what if the nativeAssetId undefined? can we make sure the nativeAssetId always have value

Reply:
It's not something I should guarantee, it's something the chain should guarantee，If it fails, the corresponding lp token will not be calculated correctly, e.g. LP-KSM/HKO

Reply:
Alright. If there is some problem with the lp-token price, how do we figure out the real issue case, does it result from this error. or should we make a conversation with the chain side when this will happen or if it happened what should we do